modules/harvest/src/Entity/HarvestRunRepository.php

```diff
@@ -243,16 +243,19 @@ public function getExtractedUuids(string $plan_id, string $run_id): array {
    *
    * @param string $plan_id
    *   Plan ID.
-   * @param string $run_id
-   *   Run ID, which is a timestamp.
+   * @param string $timestamp
+   *   The timestamp for the run. Formerly the id.
    *
    * @return \Drupal\harvest\HarvestRunInterface|\Drupal\Core\Entity\EntityInterface|null
    *   The loaded entity or NULL if none could be loaded.
+   *
+   * @deprecated in dkan:2.19.11 and is removed from dkan:3.0.0 Use HarvestService::load().
```

Comment:
It is possible that this can't be deprecated.   Everywhere it is called, there is no access to the new  actual ID.